src/main/java/seedu/address/model/travelplanner/ModelManager.java

```diff
@@ -325,9 +325,7 @@ public boolean equals(Object obj) {
         // state check
         ModelManager other = (ModelManager) obj;
         return travelPlanner.equals(other.travelPlanner)
-                && userPrefs.equals(other.userPrefs)
-                && filteredTravelPlans.equals(other.filteredTravelPlans)
-                && filteredWishlist.equals(other.filteredWishlist);
+                && userPrefs.equals(other.userPrefs);
```

Comment:
Not too sure if this is okay, removing it resolves execute_validIndexFilteredList_success in DeleteFriendCommandTest

Reply:
This is resolved in the most updated version of this PR.